npm-shrinkwrap.json

```diff
@@ -12835,12 +12644,6 @@
       "integrity": "sha1-7GIRvtGSBEIIj+D3Cyg3Iy7SyKg=",
       "dev": true
     },
-    "sigmund": {
```

Comment:
I did an `npm run shrink` for another PR and saw the exact same set of dependencies removed and I thought I did it wrong. Not any more.